docs/config/stack.md

```diff
@@ -0,0 +1,22 @@
+## Stack Configuration
+
+
+### Function Config
+
+Functions can be configured with memory and timeout requirements per function, a default can also be set for all functions in a stack that do not have their own configuration.
+
+**Example**
+
+```yaml
+functions:
+    default:
+        # Set functions to have 512 MB of memory
+        memory: 512
+        # Set functions to have a timeout of 30 seconds
+        timeout: 30
+    # Set the memory and timeout for a specific function
+    functions/hello-world.ts:
+        memory: 1024
+        timeout: 15
+```
```

Comment:
It's probably worth noting this config goes in the `nitric.yaml` file (and/or show that in the example).

Also, if wildcards work in the function names e.g. `functions/hello-*.ts` can we document that too? If they're not supported, we should probably create a new issue to consider adding that support.

Reply:
cool, will add that note. At the moment this configuration actually goes in the nitric-*.yaml stack file, but we could extend this to the base file and add inheritance for the stack files. The assumption being that not all clouds would have the same infrastructure requirements.